examples/http_get_bearssl/http_get_bearssl.c

```diff
@@ -201,6 +229,15 @@ void http_get_task(void *pvParameters)
         br_sslio_init(&ioc, &sc.eng, sock_read, &fd, sock_write, &fd);
         printf("done.\r\n");
 
+        /* FIXME: set date & time using epoch time precompiler flag for now */
+        some_time = CONFIG_EPOCH_TIME + (xTaskGetTickCount()/configTICK_RATE_HZ);
+        xc.days = (some_time / CLOCK_SECONDS_PER_DAY) + 719528;
+        xc.seconds = some_time % CLOCK_SECONDS_PER_DAY;
```

Comment:
@UncleRus I extended the timer code to [continuously progress time](https://github.com/SuperHouse/esp-open-rtos/pull/340/commits/c060a9c82a9558a51221b43ae9e9843f6d9b3bcd#diff-6314773a2e2994cb71e821eebc6bad48R233). From a functional perspective I believe the code is ready to be merged now.